py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -845,6 +845,259 @@ def aten_ops_isinf(
     )
 
 
+@dynamo_tensorrt_converter(torch.ops.aten.add.Tensor)
+def aten_ops_add(
+    network: TRTNetwork,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.elementwise.add(
+        network,
+        target,
+        SourceIR.ATEN,
+        name,
+        args[0],
+        args[1],
+    )
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.mul.Tensor)
+def aten_ops_mul(
+    network: TRTNetwork,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.elementwise.mul(
+        network,
+        target,
+        SourceIR.ATEN,
+        name,
+        args[0],
+        args[1],
+    )
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.maximum.default)
+def aten_ops_max(
+    network: TRTNetwork,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.elementwise.max(
+        network,
+        target,
+        SourceIR.ATEN,
+        name,
+        args[0],
+        args[1],
+    )
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.minimum.default)
+def aten_ops_min(
+    network: TRTNetwork,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.elementwise.min(
+        network,
+        target,
+        SourceIR.ATEN,
+        name,
+        args[0],
+        args[1],
+    )
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.sub.Tensor)
+def aten_ops_sub(
+    network: TRTNetwork,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.elementwise.sub(
+        network,
+        target,
+        SourceIR.ATEN,
+        name,
+        args[0],
+        args[1],
+    )
+
+
+# TODO: keep this or line 54...?
+# @dynamo_tensorrt_converter(torch.ops.aten.div.Tensor)
+# def aten_ops_div(
+#     network: TRTNetwork,
+#     target: Target,
+#     args: Tuple[Argument, ...],
+#     kwargs: Dict[str, Argument],
+#     name: str,
+# ) -> Union[TRTTensor, Sequence[TRTTensor]]:
+#     return impl.elementwise.div(
+#         network,
+#         target,
+#         SourceIR.ATEN,
+#         name,
+#         args[0],
+#         args[1],
+#     )
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.pow.Tensor_Tensor)
+def aten_ops_pow(
+    network: TRTNetwork,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.elementwise.pow(
+        network,
+        target,
+        SourceIR.ATEN,
+        name,
+        args[0],
+        args[1],
+    )
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.floor_divide.default)
```

Comment:
I believe this also supports [`torch.ops.aten.floor_divide.Scalar`](https://github.com/pytorch/pytorch/blob/eafc05887f44ed47e0631506d2654c429bb91fb3/aten/src/ATen/native/native_functions.yaml#L2686-L2688). You can stack that decorator here as well.

Reply:
Thanks! but when writing tests for `aten.floor_divide.Scalar`, I tried:
```
@parameterized.expand(
        [
            ("2d", (2, 1), 1.0),
            ("3d", (2, 1, 2), 2.0),
        ]
    )
    def test_floor_div_scalar(self, _, shape, scalar):
        class floor_div(nn.Module):
            def forward(self, lhs_val):
                return torch.floor_divide(lhs_val, torch.tensor(scalar))  # also tried directly using `scalar`

        inputs = [torch.randn(shape)]
        self.run_test(
            floor_div(),
            inputs,
            expected_ops={torch.ops.aten.floor_divide.Scalar},
        )
```
but got error: `AssertionError: False is not true : expected ops {<OpOverload(op='aten.floor_divide', overload='Scalar')>}, actuall ops {<OpOverload(op='aten.floor_divide', overload='default')>}`. Do you know in what scenarios we use `aten.floor_divide.Scalar`?

Reply:
I see - that's interesting - it seems Torch traces the scalars to `default` as well. Since it is a valid schema based on that link, I think it is okay to keep the Scalar overload, but use `expected_ops={torch.ops.aten.floor_divide.default},` in the test case.